_components/button-groups/01-default.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Default
+parent: Button groups
+order: 01
+lead: The default button group arranges each button as a separate element with a gap between them. On mobile devices, the buttons are arranged vertically.
+---
+
+{% include code/preview.html component="button-groups" %}
+
+<section class="site-component-section">
+    {% include code/accordion.html component="button-groups" %}
+  <div class="usa-accordion usa-accordion--bordered site-accordion-docs">
+    <button class="usa-button-unstyled usa-accordion__button"
+        aria-expanded="true" aria-controls="default-button-groups-docs">
+      Guidance
+    </button>
+    <div id="default-button-groups-docs" aria-hidden="false" class="usa-accordion__content site-component-usage">
+      <h4>When to use the default button groups</h4>
+      <ul class="usa-content-list">
+        <li><strong>Actions have a contextual relationship.</strong> For example, the default button group can be used when a form has both a primary and alternative action.</li>
+        <li><strong>Stepping through linear content.</strong> Buttons in a button group can be used for directional navigation and actions (e.g., "Back," "Next," "Continue," "Skip," "Cancel.").</li>
+      </ul>
+      <h4>When to consider something else</h4>
+      <ul class="usa-content-list">
+        <li><strong>If actions are not related.</strong> Consider how placement and alternative structure of unrelated actions can improve usability over placing all actions in a group.</li>
+        <li><strong>When mixing destructive and non-destructive actions.</strong> This can lead to input mistakes.</li>
+        <li><strong>Linking to content.</strong> Buttons in buttons button groups should not be used for navigating through site pages.</li>
```

Comment:
@Ben-Judy-Flexion Can you clarify this one? Does this mean to discourage uses like `Next page` and `Previous page` button links?

Reply:
I think our intent is to advise against using buttons when regular ol' text links would do better. Next and Previous are okay when content is organized sequentially. We iterated on the wording of this but it might need some more clarification. Maybe:

> **Linking to content.** Buttons in buttons button groups should not be used for navigating to site pages when a text link would contextually provide a better indication of the content. Grouped buttons such as "Next" and "Previous" are okay when content is organized sequentially.

...or something like that?

Reply:
@thisisdano I think what @Ben-Judy-Flexion mentioned is what we're trying to get at.